docs-content/using.html.md.erb

```diff
@@ -36,33 +36,54 @@ If your application uses another method to access secrets, such as Conjur API ca
 
 1. Create a file named `secrets.yml`.
 
-2. Add entries to the file to define the secrets that the application will fetch. Secrets to fetch from Conjur are specified with a `!<tag>` and a pathname indicating the Conjur policy path.
+2. Add entries to the file to define the secrets that the application will
+   fetch. Secrets to fetch from Conjur are specified with a `!<tag>` and a
+   pathname indicating the Conjur policy path. More information on the
+   format for `secrets.yml` [is available below](#secrets.yml).
 
 3. Store the `secrets.yml` file in the root folder of your application.
```

Comment:
We should clarify here by adding something like

"If you deploy your application in a packaged format (eg as a JAR file), you will want to ensure your secrets.yml configuration file is available in the root application directory at deploy time (eg for a JAR file, you can store secrets.yml in the `src/main/resources` directory of your application.)"

Reply:
Noting here before I forget. I would probably also be worthwhile to add something like this to the buildpack repo README.